### PR TITLE
Fix Max Range of Simulated Scan

### PR DESCRIPTION
**Pull request by afakihcpr**
Set the value for open space and unkown to scan max_range instead of max_range + 1

Raytrace to range_max + resolution instead of range_max + 1

@efernandez @servos

### CODE REVIEW

**Comment by servos** [comment]
LGTM

File: src/ray_tracer.cpp [hunk 1]
```diff
@@ -117,24 +117,26 @@ simulateRangeScan (const nm::OccupancyGrid& grid, const gm::Pose& sensor_pose,
   {
     const double theta = scanner_info.angle_min+i*scanner_info.angle_increment;
     const gm::Point scan_max =
-      rayEndPoint(p0, theta0 + theta, scanner_info.range_max+1);
+      rayEndPoint(p0, theta0 + theta, scanner_info.range_max + grid.info.resolution);
```
Comment (efernandez) [inline src/ray_tracer.cpp:120]: Looking at https://github.com/clearpathrobotics/occupancy_grid_utils/blob/a70b91473506f924167e3c945d790a8fa692fb62/src/ray_tracer.cpp#L43, I think it's probably enough with `scanner_info.range_max`, but I'm also OK with adding the `grid.info.resolution`.

**Comment by efernandez** [comment]
:+1:

File: src/ray_tracer.cpp (hunk 1 above)
Comment (afakihcpr) [inline src/ray_tracer.cpp:120]: Yeah, It is probably enough with `range_max`  but I kept an additional safety pixel just in case.

**Pull request merged**